overrides/kubejs/server_scripts/Common/Recipes/unify_barks.js

```diff
@@ -0,0 +1,41 @@
+ ServerEvents.tags('item', event => {
+//unify tags
+    event.add('mce2:barks', ['#immersive_weathering:bark', '#farmersdelight:barks'])
+ })
+ 
+
+ ServerEvents.recipes(event => {
+    event.replaceInput(
+        {input:'#farmersdelight:barks'},
+        '#farmersdelight:barks',
+        '#mce2:barks')
+
+    event.replaceInput(
+        {input:'#immersive_weathering:bark'},
+        '#immersive_weathering:bark',
+        '#mce2:barks')
+
+//Pale Jane needs to be removed manually
+    event.remove({id:'brewinandchewin:fermenting/pale_jane_from_rice_wine'})

```

Comment:
This line isn't necessary if we make a recipe with the same ID, it just overrides it.

```suggestion
```